Sources/Core/Client/AutoCancellingSubscription.swift

```diff
@@ -34,8 +34,8 @@ import RxSwift
 ///     }
 /// }
 /// ```
-public final class Subscription {
-    fileprivate static let shared = Subscription()
+public final class AutoCancellingSubscription {
```

Comment:
We no need this class and mapping rx events to a classic way, because we have events in LLC.